docs/app-dev/indexing-transactions.md

```diff
@@ -77,19 +82,21 @@ func (app *KVStoreApplication) DeliverTx(req types.RequestDeliverTx) types.Resul
 }
 ```
 
-The transaction will be indexed (if the indexer is not `null`) with a certain attribute if the attribute's `Index` field is set to `true`. 
-In the above example, all attributes will be indexed.
+The transaction will be indexed (if the indexer is not `null`) with a certain
+attribute if the attribute's `Index` field is set to `true`. In the above example,
```

Comment:
I'd replace the parens with commas and use "when" rather than "if". 

The term "certain attribute" is confusing to me. Which attribute? How is that attribute selected. 

Location-relative language "in the above example," is difficult for the same reason that relative paths are difficult in programs: they're both fragile and increase the cognitive/context specific load for reading. I believe (and I might be wrong) but the salient factor is "when no attribute is specified to index, and the index attribute is set to true."

Reply:
Yeah the whole thing reads weird (I didn't write it). I just rephrased the entire thing haha.